CHANGELOG.md

```diff
@@ -7,20 +7,6 @@ and this project adheres to [Semantic Versioning](https://semver.org).
 
 ## [Unreleased]
 
-### Added
-
-- Improved the ARABIC function to also hande short-hand roman numerals
-
-### Fixed
-
-- Fix ROUNDUP and ROUNDDOWN for floating-point rounding error [#1404](https://github.com/PHPOffice/PhpSpreadsheet/pull/1404)
-
-## [1.11.0] - 2020-03-02
```

Comment:
Hi! Is there any reason to delete part of the changes of the version 1.11.0 from the CHANGELOG?

Reply:
No, that must have been a merging conflict. Changelogs are notorious for that.
I'm sorry for any inconvenience

Reply:
Indeed that was a mistake solved in 5daa38f4563a63389593a4cd10bdafc8336509b5